rhino/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.1"
+
+services:
+  pilot-4-podman-rhino:
+    build:
+      context: .
+      dockerfile: ../Dockerfile
```

```suggestion
      context: ..
```

You won't believe the fix for this :)

---

🤯